native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -485,14 +489,19 @@ static apr_status_t create_worker(proxy_server_conf *conf, proxy_balancer *balan
             }
             return APR_SUCCESS; /* Done Already existing */
         }
-        ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, server,
-                     "Created: can't reuse worker as it for %s cleaning...", url);
+        ap_log_error(APLOG_MARK, APLOG_ERR, 0, server, SAMEURL, url);
```

Comment:
Error message for the misconfiguration.
Debug message monitoring the ntotal/nidle.